app/src/Home/index.js

```diff
@@ -0,0 +1,12 @@
+import stockGif from "./stocks.gif";
+import styles from "./styles.module.scss";
+
+const Home = () => (
+  <section className={styles.home}>
+    <h1>Welcome to Stock Hacks!</h1>
+    <p>This is a new-investor-friendly blog to learn about stock market 101</p>
+    <img src={stockGif} alt="stock list" />
```

Comment:
[suggest] I would either be more descriptive in the `alt` text, or ditch it altogether if the image is just decorative. By doing `alt=""`, screenreaders will skip the image.